### ISSUE TITLE
Feature Request: Copy to clipboard

### ISSUE THREAD

**Issue by MikeCraig418** [nalingia/nova-secret-field#2]
It would be terrific if I could copy my secret to a clipboard. The user will most likely want to do this in all of my use cases.

This can be an option on the field, like:

```
NovaSecretField::make('API Key', 'api_key')->sendsToClipboard(),
```

**Comment by nalingia** [comment]
Hi @949mac,
this is a cool feature request. I'm going to work on it after fixing bugs #1 and #3.

It will probably be available on a minor release.

**Comment by nalingia** [comment]
The feature is available on version 1.1.0.
